include/UIChar.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef UICHAR_HPP
+# define UICHAR_HPP
+
+# include <string>
+
+class UIChar {
+
+	public:
+		UIChar(std::string const &);
+		UIChar(UIChar const &) = delete;
+		UIChar(UIChar &&) = delete;
+		UIChar &operator=(UIChar const &) = delete;
+		UIChar &operator=(UIChar &&) = delete;
+		virtual ~UIChar(void) = default;
+		
+		std::string getName(void) const;
+		void setScore(int);
```

Comment:
Eeeeh, yep, true. But as i said, UIChar is not finished cause we have to talk about it and how we're going to transfer the data between the game itself and the ui.